docs/source/bond_family_guide/sysadmin_guide/configuration.rst

```diff
@@ -0,0 +1,225 @@
+..
+   INTEL CONFIDENTIAL
+   Copyright(c) 2016 Intel Corporation All Rights Reserved.
+
+   The source code contained or described herein and all documents related to
+   the source code ("Material") are owned by Intel Corporation or its suppliers
+   or licensors. Title to the Material remains with Intel Corporation or its
+   suppliers and licensors. The Material contains trade secrets and proprietary
+   and confidential information of Intel or its suppliers and licensors. The
+   Material is protected by worldwide copyright and trade secret laws and
+   treaty provisions. No part of the Material may be used, copied, reproduced,
+   modified, published, uploaded, posted, transmitted, distributed, or
+   disclosed in any way without Intel's prior express written permission.
+
+   No license under any patent, copyright, trade secret or other intellectual
+   property right is granted to or conferred upon you by disclosure or delivery
+   of the Materials, either expressly, by implication, inducement, estoppel or
+   otherwise. Any license under such intellectual property rights must be
+   express and approved by Intel(R) in writing.
+
+*************
+Configuration
+*************
+
+Overview
+========
+
+This article guides you through the configuration and startup of the Sawtooth
+Lake Bond Family validator service that you installed in :doc:`installation`.
+
+
+Prerequisites
+=============
+
+Networking
+----------
+
+1. Your node must be able to communicate with other nodes in the network, as
+covered in the networking section of :doc:`installation`.
+
+2. You need information in the form of an email document from your Intel
+representative, which contains the necessary  values for your NodeName and
+LedgerURL entries.
+
+
+Step 1: Customize the Config File for Your Environment 
+======================================================
+
+The configuration file, ``txnvalidator.js``, is located in the folder with the
+path ``C:\Program Files (x86)\Intel\sawtooth-validator\conf``.  This file has a
```

Comment:
This was correct when the documentation was created, verified by multiple installs. 
@rbuysse  has this been changed?